install/kots/manifests/kots-config.yaml

```diff
@@ -253,7 +263,7 @@ spec:
           help_text: A self-signed certficate should only be used if applying TLS termination to your load balancer or other proxy.
 
         - name: cert_manager_enabled
-          title: Use cert-manager
+          title: Use existing cert-manager
```

Comment:
Also, I think we have to update this section. Not enabling this does not mean we aren't using cert-manager right? It means we are only using it for internal communication.

cc: @MrSimonEmms